docs/model-dev-guide/dtrain/dtrain-introduction.rst

```diff
@@ -8,93 +8,93 @@
  How Determined Distributed Training Works
 *******************************************
 
-Distributed training in Determined utilizes data-parallelism. Data-parallelism for deep-learning
-consists of a set of workers, where each worker is assigned to a unique compute accelerator such as
-a GPU or a TPU. Each worker maintains a copy of the model parameters (weights that are being
-trained), which is synchronized across all the workers at the start of training.
+Determined employs data parallelism in its approach to distributed training. Data parallelism for
+deep learning consists of a set of workers, where each worker is assigned to a unique compute
+accelerator such as a GPU or a TPU. Each worker maintains a copy of the model parameters (weights
+that are being trained), which is synchronized across all the workers at the start of training.
```

Comment:
Not sure it's worth noting in this document, but this is only one of several parallelism strategies that the user can employ.

Reply:
Determined also uses deep speed, for example
we can ignore this comment